### PR TITLE
repology: fix homepage for a few packages - nothing else

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

repology was showing a few errors for packages I maintain - think of it as spring cleaning.

No other changes.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by peterhoeg** [comment]
@GrahamcOfBorg build puddletag krename anydesk broadlink-cli

**Review by dotlambda — CHANGES_REQUESTED** [review]
"broadlink" -> "pythonPackages.broadlink"

File: pkgs/applications/misc/krename/default.nix [hunk 1]
```diff
@@ -17,15 +17,17 @@ in mkDerivation rec {
     sha256 = "136j1dkqrhv458rjh5v3vzjhvq6dhz7k79zk6mmx8zvqacc7cq8a";
   };
 
+  buildInputs = [ taglib exiv2 podofo ];
+
+  nativeBuildInputs = [ extra-cmake-modules kdoctools wrapGAppsHook ];
+
+  propagatedBuildInputs = [ kconfig kcrash kinit kjsembed ];
+
```
Comment (matthiasbeyer) [inline pkgs/applications/misc/krename/default.nix]: No other changes, eh?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
These are not only homepage link fixes.

File: pkgs/applications/misc/krename/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/applications/misc/krename/default.nix]: Indeed, "cleanup" might be a better commit message

Comment (matthiasbeyer) [reply pkgs/applications/misc/krename/default.nix]: Not even that, there are new dependencies imported in `krename`. I would split that out and explain why in the commit message.

For the "update homepage" parts of the patchset, I'd approve... but merging stuff with the wrong commit message is a no-no for me.

:smile:

Comment (dotlambda) [reply pkgs/applications/misc/krename/default.nix]: > Not even that, there are new dependencies imported in krename. I would split that out and explain why in the commit message.

There aren't. The lines are just moved.

Comment (matthiasbeyer) [reply pkgs/applications/misc/krename/default.nix]: Oh, that's actually true. Sorry about that! 

Still, the commit message should be more explanatory about that ("Moved some attributed for better readability" would be fine for me, I guess).

Comment (peterhoeg) [reply pkgs/applications/misc/krename/default.nix]: Moving a few lines up before the meta tag doesn't change anything in terms of input (and hence output). But fair enough, I have amended the commit to mention the minor style changes.

**Comment by peterhoeg** [comment]
@dotlambda, good catch on the name - fixed.

**Pull request merged**